test/testdata/gofactory.go

```diff
@@ -0,0 +1,32 @@
+//golangcitest:args -Egofactory
+package testdata
+
+import (
+	"net/http"
+	alias_blocked "net/http"
+)
+
+type Struct struct{}
+
+var (
+	defaultGlobalRequest    = http.Request{}  // want `Use factory for http.Request`
+	defaultGlobalRequestPtr = &http.Request{} // want `Use factory for http.Request`
+)
+
+func Default() {
+	_ = http.Request{}  // want `Use factory for http.Request`
+	_ = &http.Request{} // want `Use factory for http.Request`
+
+	_ = []http.Request{{}, http.Request{}}   // want `Use factory for http.Request`
+	_ = []*http.Request{{}, &http.Request{}} // want `Use factory for http.Request`
+
+	call(http.Request{}) // want `Use factory for http.Request`
+
+	_ = []Struct{{}, {}}
+}
+
+func call(_ http.Request) {}
+
+func alias() {
+	_ = alias_blocked.Request{} // want `Use factory for http.Request`
```

Comment:
cosmetic question: why not `Use factory for alias_blocked.Request`?
at fist time I was confused, but cannot decide what option is better 🤔

Reply:
Idk too)
Could you decide how to better based on your experience?

I checked [wrapcheck](https://github.com/tomarrell/wrapcheck), [go-reassign](https://github.com/curioswitch/go-reassign/tree/main). They both used different printing way for package names.
I don't find another linter which uses package names.

wrapcheck: `error returned from external package is unwrapped: sig: func encoding/json.Marshal(v any) ([]byte, error)`
go-reassign: `reassigning variable EOF in other package alias`

Reply:
IMHO, the option that is better for grep and common understanding – to keep the source code and warnings for it consistent:

```go
_ = neturl.URL{}  // want `Use factory for neturl.URL`
_ = &url.URL{}    // want `Use factory for url.URL`
```